tools/topology/CMakeLists.txt

```diff
@@ -80,6 +80,7 @@ set(TPLGS
 	"sof-tgl-nocodec\;sof-tgl-nocodec"
 	"sof-tgl-rt1308\;sof-tgl-rt1308"
 	"sof-tgl-rt1308-nohdmi\;sof-tgl-rt1308-nohdmi"
+	"sof-imx8qxp-nocodec\;sof-imx8qxp-nocodec\;-DPLATFORM=imx8"
```

Comment:
why do you need this define? You are not testing against it in the topology, or I missed it?

Reply:
@plbossart oh, I see. I blindly followed the approach from other platfs.

Reply:
@plbossart fixed